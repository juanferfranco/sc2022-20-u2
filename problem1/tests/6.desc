Adiciona 3 eventos los borra todos y luego adiciona 1
