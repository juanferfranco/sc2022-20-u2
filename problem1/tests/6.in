add
ev1
add
ev2
add
ev3
ask
rm
ev1
rm
ev2
rm
ev3
ask
add
ev1
ask
exit
