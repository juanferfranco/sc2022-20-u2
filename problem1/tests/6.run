./main < tests/6.in
