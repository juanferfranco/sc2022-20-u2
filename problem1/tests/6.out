ev1
ev2
ev3
empty
ev1
